gov_uk_dashboards/components/plotly/apply_and_reset_filters_buttons.py

```diff
@@ -0,0 +1,25 @@
+"""apply_and_reset_filters_buttons"""
+from dash import html
+
+
+def apply_and_reset_filters_buttons():
+    """
+    Return clear all selections link which is aligned to the right
+    """
+    return html.Div(
+        [
+            html.A(
+                "Reset filters",
+                href="?",
+                className="govuk-link govuk-body",
+            ),
+            html.Button(
+                "Apply filters",
+                id="submit-button",
+                n_clicks=0,
```

Comment:
Do we need this `n_clicks` attribute?

Reply:
We use the n_clicks to trigger the callbacks, it's what they use in the plotly docs and we couldn't see an attribute which made more sense, even though we don't need to do anything with the number